content/en/docs/instrumentation/js/serverless/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Serverless
+aliases: [/docs/js/serverless]
```

Comment:
This alias should be dropped. The other similar aliases that are present in older pages are there for historical reasons. We don't create new aliases to pages to the non-existent section `/docs/js`.

Reply:
Thanks for letting me know! dropped.